analyses/cms-open-data-ttbar/config.yaml

```diff
@@ -0,0 +1,82 @@
+global:
+
+  # ServiceX: ignore cache with repeated queries
+  SERVICEX_IGNORE_CACHE: false
+  
+  # analysis facility: set to "coffea_casa" for coffea-casa environments, "EAF" for FNAL, "local" for local setups
+  AF: coffea_casa
+  
+benchmarking:
+
+  # chunk size to use
+  CHUNKSIZE: 200000
```

Comment:
The decrease is only needed for the ML part, right? Here we're still fine with 500k?